next-is.js

```diff
@@ -522,6 +522,84 @@ is.js may be freely distributed under the MIT Licence.
           return !!ua.match(new RegExp(inAppBrowserDetectionRules , 'ig'));
         }
       }
+    },
+    isWindowsTabletAsync: function() {
```

Comment:
Why is this async?

Reply:
because it relies on `navigator.mediaDevices.enumerateDevices()` that is async too